nbs/docs/getting-started/05_datarequirements.ipynb

```diff
@@ -80,13 +80,15 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "100%|██████████| 1.76M/1.76M [00:00<00:00, 5.55MiB/s]\n",
-      "INFO:datasetsforecast.utils:Successfully downloaded M3C.xls, 1757696, bytes.\n"
+      "100%|█████████████████████████████████████████████████████████████████████████████████████████████| 1.76M/1.76M [00:00<00:00, 25.2MiB/s]\n",
```

Comment:
Non-blocking but nice to remove this cell output

Reply:
Removed